massa-execution-worker/src/interface_impl.rs

```diff
@@ -613,6 +615,14 @@ impl Interface for InterfaceImpl {
             Slot::new(validity_start.0, validity_start.1),
             Slot::new(validity_end.0, validity_end.1),
             data.to_vec(),
+            filter
+                .map(|(addr, key)| {
+                    Ok::<AsyncMessageTrigger, ModelsError>(AsyncMessageTrigger {
+                        address: Address::from_str(addr)?,
+                        datastore_key: key.map(|k| k.to_vec()),
```

Comment:
We should test for some max length here to avoid allocating too much memory here?

Reply:
Good catch ! Done.